<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gerer_emploi</class>
 <widget class="QDialog" name="gerer_emploi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Heure de debut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="afficher">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(85, 170, 255);
background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>afficher Emploi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retour">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="heure_deb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>471</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="modifier">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="inserer">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>270</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saisir ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Heure de sortie </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="heure_sort">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Heure de pause </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="heure_pause">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Duree de pause </string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="duree_pause">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>                                     Emploi du temps </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="supprimer">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>350</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
